### ISSUE TITLE
Remove old content folder

### ISSUE THREAD

**Issue by pieterlukasse** [cBioPortal/cbioportal#768]
I think that the items in folder 
https://github.com/cBioPortal/cbioportal/tree/rc/portal/src/main/resources/content
can be cleaned-up. 
@jjgao : can you confirm?

**Comment by jjgao** [comment]
There are still a few useful ones such as examples, OQL, & web apis. Maybe we should move all of them to git?

**Comment by pieterlukasse** [comment]
yes, maybe for consistency it would be better to move all indeed.

**Comment by pieterlukasse** [comment]
@jjgao any updates on this one?

**Comment by jjgao** [comment]
@pieterlukasse I think there are still some such as the old web api page. I added it to #4837 -- we'll definitely need to get it down when we retire portal module.

**Comment by pieterlukasse** [comment]
ok. I'll close this one since it is tracked in #4837 now

**Comment by jjgao** [comment]
I'll keep it open since #4837 refers to this one.
